src/ModelerApp.vue

```diff
@@ -36,7 +36,17 @@ import {
 } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
 
-import InitialControls from "./controls";
+// Our initial node types
```

Comment:
The variable name `initialNodes`, to me, indicates that this list might be changed (like a "default value"), but it seems to remain constant. Would a more appropriate name be `nodes`?

I try to avoid mixing ES module and CommonJS syntax; maybe this list could be moved into `/src/components/nodes/index.js` and exported as an array from there?